### DISCUSSION TITLE
Update theme colors on v4

### DISCUSSION THREAD

**Discussion by jongbonga** [ColorlibHQ/AdminLTE#5656] [Q&A]
on v3 I changed the them with 

```
$theme-colors: (
  "warning": #xxxxxx
  "primary": #xxxxxx,
  "tertiary": #xxxxxx,
  etc
);
```

however, this doesn't work on v4. How can I proceed?

**Comment by danny007in** [comment]
Please check out https://getbootstrap.com/docs/5.3/customize/color/#generating-utilities
